### PR TITLE
Cherry-pick #21196 to 7.x: Fix: Dissect Cisco ASA 302013 message usernames

### PR DESCRIPTION
**Pull request by andrewkroh**
Cherry-pick of PR #21196 to 7.x branch. Original message: 

Fixes ingest pipeline filebeat-7.9.1-cisco-asa-asa-ftd-pipeline to dissect messages of the following format (optional usernames):
Built inbound TCP connection 27215708 for internet:10.2.3.4/49926 (1.2.3.4/49926)(LOCAL\username) to vlan-42:1.2.3.4/80 (1.2.3.4/80) (username)\n

Currently one can only dissect this format:
Built inbound TCP connection 27215708 for internet:10.2.3.4/49926 (1.2.3.4/49926) to vlan-42:1.2.3.4/80 (1.2.3.4/80)\n

### CODE REVIEW

**Comment by andrewkroh** [comment]
run tests

**Review by marc-gr — APPROVED** [review]

File: x-pack/filebeat/module/cisco/asa/test/sample.log [hunk 1]
```diff
@@ -38,7 +38,6 @@ Apr 30 2013 09:23:43: %ASA-5-106100: access-list acl_in est-allowed tcp inside/1
 Apr 30 2013 09:23:43: %ASA-5-106100: access-list acl_in est-allowed tcp inside/10.0.0.16(2013) -> outside/192.0.0.89(2000) hit-cnt 1 first hit [0x71a87d94, 0x0]
 Apr 15 2018 09:34:34 EDT: %ASA-session-5-106100: access-list acl_in permitted tcp inside/10.0.0.16(2241) -> outside/192.0.0.99(2000) hit-cnt 1 first hit [0x71a87d94, 0x0]
 Dec 11 2018 08:01:24 <IP>: %ASA-6-302015: Built outbound UDP connection 447235 for outside:192.168.77.12/11180 (192.168.77.12/11180) to identity:10.0.13.13/80 (10.0.13.13/80)
-Dec 11 2018 08:01:24 <IP>: %ASA-6-302015: Built outbound UDP connection 447235 for outside:192.168.77.12/11180 (192.168.77.12/11180) to identity:10.0.13.13/80port> (10.0.13.13/80)
```
Comment (P1llus) [inline x-pack/filebeat/module/cisco/asa/test/sample.log:41]: Was this one meant to be removed?

**Review by P1llus — APPROVED** [review]
LGTM. Added one small comment, and I also saw that the new event has `cisco.asa.mapped.*` while they maybe should have been removed (as they exist with source/destination).

Seeing as this is a cherry-pick however the change itself is LGTM.

File: x-pack/filebeat/module/cisco/asa/test/sample.log (hunk 1 above)
Comment (andrewkroh) [reply x-pack/filebeat/module/cisco/asa/test/sample.log:41]: Yes, it's the same as the line above, but with a "corrupt" port value which I think was an accidental addition.

**Pull request merged**